PYTHON-MANIFEST.in

```diff
@@ -7,6 +7,7 @@ graft include/grpc
 graft third_party/boringssl
 graft third_party/nanopb
 graft third_party/zlib
+include src/python/grpcio/build.py
```

Comment:
I'd honestly love to get it renamed to something like `HACKY-HACKY-THING-BECAUSE-DISTUTILS-AND-WINDOWS.py`, but you can't import dashed module names normally.